### ISSUE TITLE
use SafeTestsets

### ISSUE THREAD

**Issue by ArnoStrouwen** [SciML/QuasiMonteCarlo.jl#104]
The tests for this package are a bit unwieldy.
Best to split them up using SafeTestsets.

**Comment by ChrisRackauckas** [comment]
Yes, there's a bit of a style break in this package which makes the CI a bit harder to maintain. We do need to do the standard splitting and safetestsets
